pubspec.yaml

```diff
@@ -1,14 +1,11 @@
 name: buildbucket
-authors:
-        - Kendal Harland <kendal@kendalhar.land>
-homepage: https://github.com/kharland/dart-buildbucket
-version: 0.3.0
-description: Auto-generated client libraries for the buildbucket API.
+version: 0.1.0-dev
```

Comment:
Maybe this should be `0.3.1`?

You might want to hand-edit this `pubspec.yaml` file (to keep `homepage` / `description` / ...)

Consider updating `CHANGELOG.md` also (since it will be published on pub)

Reply:
Done